src/utils/filterutils.tsx

```diff
@@ -84,16 +85,31 @@ function parseNumber(num: string) {
 }
 
 /**
- * Deselects the selected static number range filters in state.
+ * Deselects the selected static number range filters in state. If fieldIds are
+ * provided, only filters corresponding to one of those fieldIds are deselected.
+ * Otherwise, all selected filters are deselected.
  */
-export function clearStaticRangeFilters(answersActions: AnswersActions){
+export function clearStaticRangeFilters(answersActions: AnswersActions, fieldIds?: string[]) {
```

Comment:
maybe `getSelectedNumericalFacetFields` and the fieldIds param could be a Set instead of an array? Array is also good I don't think there's really much difference for what we're doing

Reply:
sure, I can change to Set